pkgs/development/interpreters/python/wrapper.nix

```diff
@@ -20,10 +23,16 @@ let
       fi
       mkdir -p "$out/bin"
 
-      cd "${python}/bin"
-      for prg in *; do
-        rm -f "$out/bin/$prg"
-        makeWrapper "${python}/bin/$prg" "$out/bin/$prg" --set PYTHONHOME "$out"
+      for path in ${stdenv.lib.concatStringsSep " " paths}; do
+        if [ -d "$path/bin" ]; then
+          cd "$path/bin"
+          for prg in *; do
+            if [ -f "$prg" ]; then
```

Comment:
this check if file I had to do since I noticed that environments now have a __pycache__ folder in bin. I guess since the wheels changes Python is run now somewhere leaving this behind.